src/modules/menu/components/advancedTools/ImportFile/ImportFile.vue

```diff
@@ -1,67 +1,55 @@
 <script setup>
 import { ref } from 'vue'
-import { useStore } from 'vuex'
 
 import ImportFileLocalTab from '@/modules/menu/components/advancedTools/ImportFile/ImportFileLocalTab.vue'
 import ImportFileOnlineTab from '@/modules/menu/components/advancedTools/ImportFile/ImportFileOnlineTab.vue'
-import SimpleWindow from '@/utils/components/SimpleWindow.vue'
 
 const selectedTab = ref('online')
-
-const store = useStore()
 </script>
 
 <template>
-    <teleport to="#map-footer-middle-1">
```

Comment:
I think this teleport target was added for the sole purpose of this ImportFile component, so if we do not use it anymore, that might be a good idea to clear it too (and in the process simplify the structure of the InfoboxModule)